src/front-end/src/components/Login.js

```diff
@@ -12,8 +10,6 @@ class Login extends Component {
     this.state = {
       email: '',
     };
-    this.localResetPassword = this.localResetPassword.bind(this);
-    this.handleEmailChange = this.handleEmailChange.bind(this);
```

Comment:
The localResetPassword method is used in ForgotPassword.js, so I removed it from here since it doesn't do anything in Login.js.
onChange handles both password and email changes, so I removed handleEmailChange.